Automattic/wp-calypso
# Revert plan page quick improvements copy test

#### Proposed Changes

* Revert plan page quick improvements copy test to the `control` version.
* The experiment was introduced by the PR #66696.

#### Testing Instructions



* Go through the signup flow and verify that the plans step is rendered correctly. 
* Verify that the feature list in the plans step matches the one in production for the signup flow in desktop and mobile views.
* Also verify the feature list in logged-in `/plans` page and verify that it matches the one in production, desktop and mobile view.
